android/src/main/java/com/maskedtextinput/managers/AdvancedTextInputMaskDecoratorViewManager.kt

```diff
@@ -41,34 +42,32 @@ class MaskedTextInputDecoratorViewManager(
   }
 
   @ReactProp(name = "defaultValue")
-  fun setDefaultValue(
-    view: MaskedTextInputDecoratorView,
+  override fun setDefaultValue(
```

Comment:
I think this property comes from RN? Why do you need to manage these properties?..